source/MaterialXTest/ShaderValid.cpp

```diff
@@ -994,6 +1004,20 @@ TEST_CASE("MaterialX documents", "[shadervalid]")
                         renderableSearchTimer.endTimer();
                         if (impl)
                         {
+                            if (options.checkImplCount)
+                            {
+                                // Get the nodegraph implementation name if any
```

Comment:
Not all implementations are recorded in the shadergen cache, which includes nodegraphs.